tests/OmniSharp.Roslyn.CSharp.Tests/FixUsingsFacts.cs

```diff
@@ -250,7 +250,7 @@ public method1()
                     Line = point.Line,
                     Column = point.Offset,
                     FileName = TestFileName,
-                    Text = "`classX` is ambiguous"
+                    Text = "`classX` is ambiguous. Namespaces: using nsA; using nsB;",
```

Comment:
@tadesegha This isn't your fault, but I don't think this test does what we think it does;

* I don't see any code in this test that asserts on the expected content
* When pasting this code into VS Code, I get the error "The type or namepspace ClassX could not be found" and quick fixes to add usings for either of the namespaces. 

Can you fix up the test to actually assert something? Thanks!